gh-79097: Add support for aggregate window functions in sqlite3 (#20903)
python/cpython: Misc/NEWS.d/next/Library/2020-05-24-23-52-03.bpo-40617.lycF9q.rst

> Add support for aggregate window functions in :mod:`sqlite3`. Patch by Erlend E. Aasland.

Reviewer: ```suggestion
Add :meth:`sqlite3.Connection.create_window_function` to create aggregate
  window functions in :mod:`sqlite3`. Patch by Erlend E. Aasland.
```

Author: I appreciate your comments, Kumar, but I would prefer to get them after I mark my PR ready for review 🙂 Thanks!
